src/System.Console/tests/TermInfo.cs

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Text;
+using Xunit;
+
+public class TermInfo
+{
+    [Fact]
+    [PlatformSpecific(PlatformID.AnyUnix)]
+    public void VerifyInstalledTermInfosParse()
+    {
+        string[] locations = GetFieldValueOnObject<string[]>("_terminfoLocations", null, Type.GetType("ConsolePal.TermInfo.Database"));
```

Comment:
I haven't had a chance to test since I'm WFH without a Linux Box available...hoping Jenkins tells me if everything is working or what needs to be fixed :)